### ISSUE TITLE
PIXI Graphics Color Bleed Issue

### ISSUE THREAD

**Issue by karthik947** [pixijs/pixijs#7512]
### Expected Behavior
I have two functions using same instance of Pixi Graphics to plot rectangles. The first function runs only once and paints 100 rectangles in red color and the other one is a recurring function that plots a white rectangle every 100ms. The bug here is that the  the second function continues to plot rectangles in red color, despite specifying color as white, after the first function is executed once. Code is provided below. I posted this as a query on pixijs forum (link below) and I was advised to report this as bug.
https://www.html5gamedevs.com/topic/47342-pixi-graphics-color-bleed-issue/ 

### Current Behavior
The color bleeds from first function to the second.

### Possible Solution
The color should not bleed from the first function to the second, even though they use the same instance of Pixi Graphics

### Steps to Reproduce
https://www.pixiplayground.com/#/edit/NIRPakxdI9WDLourp3m15

### Environment

- **`pixi.js` version**: _e.g. 4.7.1_
- **Browser & Version**: _e.g. Chrome 67_
- **OS & Version**: _e.g. Ubuntu 18.04_
- **Running Example**: _url_

**Comment by bigtimebuddy** [comment]
Maybe a duplicate of #7053?

**Comment by karthik947** [comment]
I looked into the other bug and it looks different. For now, the only solution I can see  if to use a different instance of graphics object, which fixes the issue. If there is an alternate efficient way to render graphics at high frequency, am open to it. Thank you.

**Comment by bigtimebuddy** [comment]
Multiple Graphics is probably more efficient because it's less likely to break batching. Also, consider sharing the Graphics geometry between Graphics objects if you're repeating the same thing over and over. That will keep from triangulating unnecessarily.

**Comment by bigtimebuddy** [comment]
I wonder if what is happening here is a Graphic goes from being batchable to being unbatchable and chaos ensues. Thoughts @GoodBoyDigital?
